### DISCUSSION TITLE
guide/markdown/extensions/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#261] [docs-comment]
# guide/markdown/extensions/

https://theme-plume.vuejs.press/guide/markdown/extensions/

**Comment by Jackpanger** [comment]
可以问一下::: note展示出来的效果为什么和上面不一样呀？是灰色的注

**Reply by pengzhanbo** [reply]
`::: note` 的效果就是 灰色背景的 `注`，UI 没错的。note 和 info 本质上级别是相同的，note 更偏向于 注释，稍轻量于 info

**Reply by Jackpanger** [reply]
感谢您的回复。因为看到这篇文章里::: note 的前面图标效果是绿色的，注的字体也不太一样。看您的源码里的`扩展.md`文件也没有找到相关设置，好奇您是怎么做到的？

**Reply by pengzhanbo** [reply]
绿色图标那个是 `::: tip` 吧

**Reply by Jackpanger** [reply]
我把它截屏了，`::: note
This is a note box
:::`和`> [!NOTE]
> 强调用户在快速浏览文档时也不应忽略的重要信息`。这两个展示的注都是绿色的https://github.com/Jackpanger/jackpanger.github.io/blob/master/docs/zh/notes/guide/visa/usvisa/note.jpg

**Reply by pengzhanbo** [reply]
看了截图，我有点迷糊了，我有点不理解你的问题是什么，这里的 `:::note` 和 `> [!NOTE]` 就是等价的，图标都是 `“`，默认标题也是 `注`。 两者都是如同你给出的截图。
```md
::: note
This is a note box
:::
```
![QQ_1728582511256](https://github.com/user-attachments/assets/3987da73-8985-4e3c-b76d-1b61940e23c0)

```md
> [!NOTE]
> 强调用户在快速浏览文档时也不应忽略的重要信息。
```
![QQ_1728582538542](https://github.com/user-attachments/assets/d6db1219-d417-4bc9-9891-152823e2cc5d)

**Reply by Jackpanger** [reply]
不好意思我可能描述不清楚。我是说我私下使用的时候，注 是这个效果https://github.com/Jackpanger/jackpanger.github.io/blob/master/docs/zh/notes/guide/visa/usvisa/note2.jpg，前面的图标是灰色的，字体也是灰色的。 您的文档里是绿色的，所以想问一下您是怎么调整的https://github.com/Jackpanger/jackpanger.github.io/blob/master/docs/zh/notes/guide/visa/usvisa/note.jpg

**Reply by pengzhanbo** [reply]
把主题的版本更新到最新即可。另外更新主题后可能会有缓存问题导致新内容没生效，可以用 `npm run docs:dev-clean` 重新执行。
更新主题我建议你使用 `npx vp-update` 进行更新。

**Reply by Jackpanger** [reply]
已解决，感谢！

**Comment by Easton119** [comment]
请问下博客文章旁边有设置大纲吗

**Reply by pengzhanbo** [reply]
大纲根据文章内容中的标题自动生成。

**Reply by Easton119** [reply]
emmm,我的意思就是说博客文章的侧边没有展示大纲?或者说需要在哪里启用？

**Reply by pengzhanbo** [reply]
我不确定你指的是左侧的侧边栏，还是右侧的页内大纲。
博客文章是没有左侧的侧边栏的。页内大纲则是自动根据文章内的 `h2-h6` 自动生成，默认值生成 `h2-h3` 的。你也可以在使用 [outline](https://theme-plume.vuejs.press/config/frontmatter/basic/#outline) 修改。

**Reply by Easton119** [reply]
好的，明白了

**Comment by SuniRein** [comment]
任务列表不能正常渲染

**Reply by SuniRein** [reply]
突然发现只有firefox上有这个问题，手机端和电脑端都是

**Reply by pengzhanbo** [reply]
下个版本修复

**Comment by Soltus** [comment]
内部链接能否支持通过 page.links 获取，开发插件很需要，目前获取都是空的
```ts
/**
 * Vuepress Page
 */
type Page  ... {
/**
     * Links of the page
     */
    links: MarkdownLink[];
...
}
```
